components/03-modules/product-item/ProductItem.html

```diff
@@ -0,0 +1,72 @@
+<component
+  :is="itemTag"
+  class="product-item"
+>
+  <div class="product-item__image">
+    <alpaca-image
+      :src="productImage.dataSrc"
+      :alt="productImage.alt"
+    />
+  </div>
+  <div class="product-item__info">
+    <div class="product-item__data">
+      <router-link
+        :to="href"
+        class="link link--inverted product-item__name"
+      >
+        {{ productName }}
+      </router-link>
+      <alpaca-price
+        :price="productPrice.price"
+        :special-price="productPrice.specialPrice"
+        :old-price="productPrice.oldPrice"
+        class="product-item__price-wrapper"
+        old-price-class="product-item__price-wrapper--old"
+        special-price-class="product-item__price-wrapper--special"
+      />
+      <alpaca-description-list
+        v-if="productOptions"
+        :elements="productOptions"
+        class="product-item__attributes"
+      />
+    </div>
+    <div
+      :class="[
+        'product-item__actions',
+        { 'product-item__actions--to-right': !quantity }
+      ]"
+    >
+      <div
+        v-if="quantity"
+        class="product-item__qty-wrapper"
+      >
+        <span class="product-item__qty-text">
+          {{ quantity.text }}
+        </span>
+        <alpaca-quantity-update
+          :input="quantity.input"
+          :minus-button="quantity.minusQtyButton"
+          :plus-button="quantity.plusQtyButton"
+          decrement-button-class="product-item__qty-button"
```

Comment:
IMO will be better to use minus or decrement in all places, not both. The same in case of plus/increment :D